index.js

```diff
@@ -4,6 +4,17 @@ const FormData = require("form-data")
 const isnumeric = require("isnumeric")
 const fs = require("fs")
 
+const versionReg = /(.*?)-(stable|beta|alpha|private|demo)$/gi
+
+function getVersion(version){
+	let result;
+	if ((result = versionReg.exec(version)) !== null) {
```

Comment:
I don't really like this. You already create result on line 10, is there a reason for not calling .exec on the same line, and checking for null against result on 11?